composer.json

```diff
@@ -14,7 +14,8 @@
     }
   ],
   "require": {
-    "ml/json-ld": "^1.0"
+    "ml/json-ld": "^1.0",
+    "cache/cache": "^0.3.0"
```

Comment:
Isn’t this loading all cache strategies? Should’t we just add APCu cache here?

Reply:
True, only the required caches are loaded in the most recent commit